packages/prediction/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@pancakeswap/prediction",
+  "version": "0.0.1",
+  "sideEffects": false,
+  "private": true,
+  "scripts": {
+    "test": "vitest --run"
+  },
+  "type": "module",
+  "peerDependencies": {
+    "@pancakeswap/sdk": "workspace:*"
+  },
+  "dependencies": {
+    "@pancakeswap/chains": "workspace:*",
+    "@pancakeswap/tokens": "workspace:*",
+    "@pancakeswap/pools": "workspace:*",
+    "@pythnetwork/pyth-evm-js": "^1.29.2",
+    "@tanstack/react-query": "^4.28.0",
```

Comment:
the sdk shouldn't be relying on usequery. Better practice would be sdk handling the request logic and let the web application to deal with hooks

Reply:
done